[2023-05-11T17:52:24.006+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:871: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti: TaskInstance | None = qry.with_for_update().one_or_none()

[2023-05-11T17:52:24.075+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_process manual__2023-05-11T17:52:19.505720+00:00 [queued]>
[2023-05-11T17:52:24.122+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: job_1_tranvietcuong.file_process manual__2023-05-11T17:52:19.505720+00:00 [queued]>
[2023-05-11T17:52:24.123+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:52:24.124+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-05-11T17:52:24.124+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-11T17:52:24.204+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): file_process> on 2023-05-11 17:52:19.505720+00:00
[2023-05-11T17:52:24.221+0000] {standard_task_runner.py:54} INFO - Started process 661 to run task
[2023-05-11T17:52:24.234+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'job_1_tranvietcuong', 'file_process', 'manual__2023-05-11T17:52:19.505720+00:00', '--job-id', '189', '--raw', '--subdir', 'DAGS_FOLDER/tranvietcuong.py', '--cfg-path', '/tmp/tmp5c5seo9q']
[2023-05-11T17:52:24.237+0000] {standard_task_runner.py:83} INFO - Job 189: Subtask file_process
[2023-05-11T17:52:24.240+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/tranvietcuong.py
[2023-05-11T17:52:24.475+0000] {task_command.py:384} INFO - Running <TaskInstance: job_1_tranvietcuong.file_process manual__2023-05-11T17:52:19.505720+00:00 [running]> on host 09877495cfa0
[2023-05-11T17:52:24.485+0000] {warnings.py:110} WARNING - /usr/local/lib/python3.7/site-packages/***/models/taskinstance.py:873: SAWarning: TypeDecorator ExecutorConfigType() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ti = qry.one_or_none()

[2023-05-11T17:52:24.742+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=job_1_tranvietcuong
AIRFLOW_CTX_TASK_ID=file_process
AIRFLOW_CTX_EXECUTION_DATE=2023-05-11T17:52:19.505720+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-11T17:52:19.505720+00:00
[2023-05-11T17:52:24.905+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-11T17:52:24.953+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=job_1_tranvietcuong, task_id=file_process, execution_date=20230511T175219, start_date=20230511T175224, end_date=20230511T175224
[2023-05-11T17:52:25.047+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-11T17:52:25.159+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
